### ISSUE TITLE
Improve accessibility and usability to engage a wider audience

### ISSUE THREAD

**Issue by lukefretwell** [GetDKAN/dkan#2218]
Per the Civic Analytics Network's '[An Open Letter to the Open Data Community](http://datasmart.ash.harvard.edu/news/article/an-open-letter-to-the-open-data-community-988)':

1. **Improve accessibility and usability to engage a wider audience.** Historically, the features and services on open data portals targeted more technical users, in particular developers, as early adopters. Across our cities, we see the need to broaden the number and types of users engaging open data. As a result, the features and UI design of data portals need to evolve to align with the needs and abilities of the broader public. Key features and recommendations that can guide this principle include the following:

- Portals should have simple and fast data downloads coupled with more user-friendly design for understanding data.
- Accessibility and usability changes should be data-driven; that is, reflective of what datasets and features are being used most.
- Portals should include more space for conversations on the use of, and resources for, datasets including user guides, dashboards, and social media communication.
- Portals should be exemplars of good web design (e.g., many portals are not mobile friendly).
- User research from providers (who is using my data - where are the gaps?), including strategies for marketing open data to populations that might not be using it, would be helpful. This might include additional applications or view options for those communities.
- Portals should include more intuitive ways of visualizing and exploring data. It’s time for open data to move beyond spreadsheets on the web.
- In addition to view totals for datasets, cities’ data portals should also publish information on which datasets are being downloaded.

**Comment by acouch** [comment]
Closing this ticket as it has not been updated recently and is not on our near-term roadmap. Please re-open if you would like to discuss further or are actively working on it.
